estate/models/estate_property.py

```diff
@@ -98,6 +98,11 @@ def _check_selling_and_expected_prices(self):
                     "You must reduce the expected price if you want to accept this offer."
                 )
 
+    @api.ondelete(at_uninstall=False)
+    def _unlink_only_status(self):
```

Comment:
This should be named `_unlink_if_<condition>` or `_unlink_except_<not_condition>` according to documentation.

```suggestion
    def _unlink_if_new_canceled(self):
```